app/helpers/job_references_helper.rb

```diff
@@ -1,19 +0,0 @@
-module JobReferencesHelper
-  def reference_request_status(reference_request, job_reference)
-    if reference_request.marked_as_complete?
-      "completed"
-    elsif reference_request.sent?
-      if job_reference.complete?
-        if job_reference.can_give_reference?
-          "received"
-        else
-          "declined"
-        end
-      else
-        "pending"
-      end
-    else
-      "created"
-    end
-  end
-end
```

Comment:
i don't understand why this helper has been removed - it clearly sows intent (status based on model) and has a test to go with it. Surely this should have been extended for the new cases?

Reply:
The reference request.status has the information about all the possible states making this helper unnecessary

Reply:
I just don't agree with this re-write. This code (and test) were written because numerous defects were found during manual testing. Removing this feels like a retrograde step, especially as this is now live and working. I don't believe there was a need to alter the references implementation as part of this change.